openapi/components/schemas/StorefrontQuoteChangeOrder.yaml

```
@@ -227,6 +227,38 @@ properties:
     description: |-
       Specifies if the quotation invoice must include interim items only.
       If this value is set to `false`, all upcoming items are included.
+  usageSettings:
+    type:
+      - 'object'
+      - 'null'
+    description: Settings for items with metered billing.
+    properties:
+      policy:
+        type: string
+        enum:
+          - reset
+          - transfer
+        description: Policy for removed items with metered billing plan.
+        x-enumDescriptions:
+          reset: Delete existing usages.
+          transfer: Transfer usages from removed plans to existing plans inside the subscription with `transfers` field.
```

nitpick: I found this sentence difficult to understand

I've made a suggestion based on my understanding. I'm not sure if it's correct 😅 
```suggestion
          transfer: Transfer usages from removed plans to existing plans that are in the same subscription, using the `transfers` field.
```